DeepNtuplizer/production/DeepNtuplizer.py

```diff
@@ -13,6 +13,10 @@
 options.register('job', 0, VarParsing.VarParsing.multiplicity.singleton, VarParsing.VarParsing.varType.int, "job number")
 options.register('nJobs', 1, VarParsing.VarParsing.multiplicity.singleton, VarParsing.VarParsing.varType.int, "total jobs")
 options.register('gluonReduction', 0.0, VarParsing.VarParsing.multiplicity.singleton, VarParsing.VarParsing.varType.float, "gluon reduction")
+options.register('release','8_0_1', VarParsing.VarParsing.multiplicity.singleton,VarParsing.VarParsing.varType.string,"release number (w/o CMSSW)")
```

Comment:
what about
```python
version = os.environ['CMSSW_VERSION'] #CMSSW_X_Y_Z
```
?

In this way you do not have to configure it yourself (and risk making mistakes)